java-frontend/src/main/java/org/sonar/java/resolve/SemanticModel.java

```diff
@@ -71,7 +69,6 @@ public static SemanticModel createFor(CompilationUnitTree tree, List<File> proje
 
   public void done(){
     ParametrizedTypeJavaType.typeSubstitutionSolver = null;
```

Comment:
I now wonder about this fix (due to https://jira.sonarsource.com/browse/SONARJAVA-1910). Maybe we can move it directly to the end of `VisitorsBridge.visitFile(...)` and remove the `done()` method here.